python3-edl/python3-edl.spec

```diff
@@ -0,0 +1,65 @@
+Name: python3-edl
+Version: 3.52.1+243
+%define commit 8f606dd1fe968371efe3ffdc2d1c7ebc7a888327
+Release: 1%{?dist}
+Summary: Qualcomm Arm SoC Emergency Download (EDL) client and diagnostic tools
+License: GPL-3.0-only
+URL: https://github.com/bkerler/edl
+Source0: https://github.com/bkerler/edl/archive/%{commit}.tar.gz
+BuildRequires: git python3-build python3-devel python3-installer python3-pip python3-setuptools python3-wheel xz-devel
+# Possibly missing dependencies: python3-exscript, python3-keystone (optional), and python3-pycryptodome
```

Comment:
I ran through the required `edl` commands for an Odin 2 build. They all worked, despite not having these missing dependencies (which are not currently packaged for Fedora).